composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -70,6 +100,15 @@ fun main(rawArgs: Array<String>) {
                             .subscribeOn(Schedulers.io())
                             .toList()
                             .flatMap {
+                                val testRunner = if (args.testRunner.isEmpty()) {
+                                    device.getTestRunnerForInstalledTestApk(testPackage)
```

Comment:
querying every device to get info that we can parse from apk looks very strange to me O_o